src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -147,7 +147,7 @@ public bool CanRetry
 
         public DateTimeOffset CreationTime { get; } = DateTimeOffset.UtcNow;
 
-        private async Task WriteHeadersAsync(HttpHeaders headers, CancellationToken cancellationToken)
+        private async Task WriteHeadersAsync(HttpHeaders headers, string cookies, CancellationToken cancellationToken)
```

Comment:
Nit: could we name the parameter something like "cookiesFromContainer"? Before reading the rest of the code, I was left wondering whether this was meant to be instead of or in addition to anything in the headers.

Reply:
Will do